api/crossmodel/client.go

```diff
@@ -37,3 +37,28 @@ func (c *Client) Offer(service string, endpoints []string, url string, users []s
 	}
 	return out.Results, nil
 }
+
+// Show returns offered remote service details for a given URL.
+func (c *Client) Show(url string) (params.RemoteServiceInfo, error) {
+	found := params.RemoteServiceInfoResults{}
+
+	err := c.facade.FacadeCall("Show", []string{url}, &found)
+	if err != nil {
+		return params.RemoteServiceInfo{}, errors.Trace(err)
+	}
+
+	result := found.Results
+	if len(result) > 1 {
```

Comment:
I was hoping to differentiate between "not found" and "something is wrong as only one was meant to come back"...

You don't think it's worth differentiating?